internal/worker/bootstrap/manifold.go

```diff
@@ -239,6 +247,11 @@ func Manifold(config ManifoldConfig) dependency.Manifold {
 				return nil, errors.Trace(err)
 			}
 
+			var serviceFactoryGetter servicefactory.ServiceFactoryGetter
```

Comment:
Can you delete the `controllerServiceFactory` and just get all calls to use the `serviceFactoryGetter` once you've got the model service factory.

Reply:
nice cleanup, done